src/meta/buildTags.tsx

```diff
@@ -66,6 +66,18 @@ const buildTags = (config: BuildTagsParams) => {
     );
   }
 
+  if (config.alternate) {
+    tagsToRender.push(
+      <link
```

Comment:
Only had a chance to do a little bit of research on this. But I think this tag is used for alternate mobile version and then also an alternate language version. I don't see any examples of both attributes being present at the same time.

I am thinking it would be easier to use if it was split into two properties. 

```
interface MobileAlternate {
  media: string;
  href: string;
}

interface LanguageAlternate {
  hrefLang: string;
  href: string;
}
```

and then we would have two tags being spit out.

Reply:
Good idea :)